docs/en/setup/backend/configuration-vocabulary.md

```diff
@@ -134,6 +134,7 @@ core|default|role|Option values, `Mixed/Receiver/Aggregator`. **Receiver** mode
 | - | -| sampleRate|Sampling rate for receiving trace. The precision is 1/10000. 10000 means 100% sample in default.|SW_TRACE_SAMPLE_RATE|10000|
 | - | - |slowDBAccessThreshold|The slow database access thresholds. Unit ms.|SW_SLOW_DB_THRESHOLD|default:200,mongodb:100|
 | - | - |forceSampleErrorSegment|When sampling mechanism activated, this config would make the error status segment sampled, ignoring the sampling rate.|SW_FORCE_SAMPLE_ERROR_SEGMENT|true|
+| - | - |exceptionAnalyzerStrategy|Define how to extract segment status.If the exceptionAnalyzerStrategy equals to ENTRY_FIRST, the final status would be follow entry span status.Otherwise,the status would be error if the status of any spans is error. |SW_EXCEPTION_ANALYZER_STRATEGY|ENTRY_FIRST|
```

Comment:
Also, notice, `ENTRY_FIRST` is not the right term. The entry span could be not the first, and vice versa.